src/go/k8s/tests/e2e-v2/resource-topic-crud/05-assert-increase-topic-replication-factor.yaml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
The names of the assert and the operation do not match. Is this intentional?

Reply:
Thank you. My mistake. Fixed in the next push.